src/features/venueMap/components/VenueMapBottomSheet/VenueMapOfferPlaylist.tsx

```diff
@@ -53,6 +53,7 @@ export const VenueMapOfferPlaylist = ({
         chroniclesCount: item.offer.chroniclesCount,
         headlineCount: item.offer.headlineCount,
         hasSmallLayout: true,
+        hasSoonOffer: item.offer.tags?.includes('is_future'),
```

Comment:
```suggestion
        isComingSoonOffer: item._tags?.includes('is_future'),
```
Je préfère le `is` vu que c'est l'état de l'offre